examples/xoxo/xoxo.py

```diff
@@ -0,0 +1,183 @@
+from pysmt.shortcuts import FreshSymbol, Symbol, String, LE, GE, Int, Not, Or, And, Equals, Plus, Solver, StrContains
+from pysmt.typing import INT, STRING
+from enum import Enum
+import logging
+logger = logging.getLogger('xoxo')
+logger.setLevel(logging.DEBUG)
+ch = logging.StreamHandler()
+ch.setLevel(logging.INFO) # change to DEBUG for info on assertions
+logger.addHandler(ch)
+
+class Cell(Enum):
+    x = 1
+    o = 10
+    s = 0
+
+DEBUG = False
```

Comment:
You can use the python's built-in __debug__ .

Reply:
I can't find any documentation for a built-in debug - can you link?

Reply:
https://docs.python.org/3/library/constants.html#__debug__

Reply:
thanks, used __debug__ to switch log levels